newIDE/app/src/Export/Builds/BuildCard.js

```diff
@@ -134,7 +144,7 @@ export const BuildCard = ({
   const [showCopiedInfoBar, setShowCopiedInfoBar] = React.useState(false);
 
   const [isEditingName, setIsEditingName] = React.useState(false);
-  const [name, setName] = React.useState(build.name || '');
+  const [name, setName] = React.useState(build.name || buildName);
```

Comment:
this can probably directly be `buildName` no? (see logic in the definition is exactly the same)

Reply:
Oh yeah you're right